integration-tests/src/test/java/oracle/kubernetes/operator/ITMultipleClusters.java

```diff
@@ -0,0 +1,212 @@
+  @AfterClass
+  public static void staticUnPrepare() throws Exception {
+    logger.info("+++++++++++++++++++++++++++++++++---------------------------------+");
+    logger.info("BEGIN");
+    logger.info("Run once, release cluster lease");
+    tearDown(new Object() {}.getClass().getEnclosingClass().getSimpleName());
+    logger.info("SUCCESS");
+  }
+
+  /**
+   * Create 2 configured clusters in a domain each having 2 managed servers. Verify the managed
+   * servers are running and verify the basic use cases.
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testCreateDomainTwoConfiguredCluster() throws Exception {
+    String DOMAINUID = "twoconfigclustdomain";
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    String template =
+        BaseTest.getProjectRoot() + "/kubernetes/samples/scripts/common/domain-template.yaml";
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    // create operator1
+    if (operator1 == null) {
+      operator1 = TestUtils.createOperator(OPERATOR1_YAML);
+    }
+    Domain domain = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> domainMap = TestUtils.loadYaml(DOMAINONPV_WLST_YAML);
+      domainMap.put("domainUID", DOMAINUID);
+      domainMap.put("clusterType", "CONFIGURED");
+      domainMap.put(
+          "createDomainPyScript",
+          "integration-tests/src/test/resources/domain-home-on-pv/"
+              + TWO_CONFIGURED_CLUSTER_SCRIPT);
+
+      String add =
+          "  - clusterName: %CLUSTER_NAME%-2\n"
+              + "    serverStartState: \"RUNNING\"\n"
+              + "    replicas: %INITIAL_MANAGED_SERVER_REPLICAS%\n";
+      logger.info("Making a backup of the domain template file:" + template);
+      if (!Files.exists(Paths.get(template + ".org"))) {
+        Files.copy(Paths.get(template), Paths.get(template + ".org"));
+      }
+      Files.write(Paths.get(template), add.getBytes(), StandardOpenOption.APPEND);
+      byte[] readAllBytes = Files.readAllBytes(Paths.get(template));
+      logger.info(new String(readAllBytes, StandardCharsets.UTF_8));
+      domain = TestUtils.createDomain(domainMap);
+      domain.verifyDomainCreated();
+      K8sTestUtils testUtil = new K8sTestUtils();
+      String domain1LabelSelector = String.format("weblogic.domainUID in (%s)", DOMAINUID);
+      String namespace = domain.getDomainNS();
+      String pods[] = {
+        DOMAINUID + "-" + domain.getAdminServerName(),
+        DOMAINUID + "-managed-server",
+        DOMAINUID + "-managed-server1",
+        DOMAINUID + "-managed-server2",
+        DOMAINUID + "-new-managed-server1",
+        DOMAINUID + "-new-managed-server2",
+      };
+      for (String pod : pods) {
+        assertTrue(
+            pod + " Pod not running", testUtil.isPodRunning(namespace, domain1LabelSelector, pod));
+      }
+      testBasicUseCases(domain);
```

Comment:
when a webapp is deployed, can you see to which cluster is targeting to ?

Reply:
It is deploying to the default cluster initialized in the domain cluster1